faststream/app.py

```diff
@@ -236,3 +238,17 @@ async def _shutdown(self, log_level: int = logging.INFO) -> None:
     def _log(self, level: int, message: str) -> None:
         if self.logger is not None:
             self.logger.log(level, message)
+
+    @property
+    def should_exit(self) -> bool:
+        """Get should exit flag state."""
+        return self._should_exit.is_set()
+
+    @should_exit.setter
+    def should_exit(self, value: Literal[True]) -> None:
+        """Set should exit flag state."""
+        if value is not True:
+            msg = "'should_exit' can receive only 'True' value"
+            raise ValueError(msg)
+
+        self._should_exit.set()
```

Comment:
I'm not sure if you want to save `should_exit` public boolean or not

Reply:
No, just remove it

Reply:
Just remove it :)